app/addons/activetasks/tests/activetasks.componentsSpec.react.jsx

```diff
@@ -70,6 +70,14 @@ define([
         window.confirm.restore && window.confirm.restore();
       });
 
+
+      it('it displays a message instead of an empty table, if there are undefined active tasks', function () {
+        spy = sinon.spy(Actions, 'setCollection');
```

Comment:
actually just leaving out this line does the trick except that we'd have to modify afterEach: 
```
        if (spy) {
          spy.restore();
          spy = undefined;
        }
```